cypress/e2e/signIn.cy.js

```diff
@@ -0,0 +1,60 @@
+/// <reference types="cypress" />
+
+describe('User should', () => {
+
+    const username = 'tomsmith';
+    
+    const password = 'SuperSecretPassword!';
+
+    it('login with valid creds', () => {
+
+    cy.visit('/login');
+    
+    cy.get('#username')
+        .type(username);
```

Comment:
method indentation should be the same
```suggestion
      .type(username);
```